src/install.rst

```diff
@@ -6,361 +6,329 @@ Installing Nextstrain
 
-         The final output from the last command should look like this, where ``<option>`` is the option chosen in the previous step:
+            3. Open a WSL terminal by running **wsl** from the Start menu.
+            4. Install the Nextstrain CLI:
 
-         .. code-block:: none
+               .. code-block:: bash
 
-            Setting default environment to <option>.
+                  curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/linux | bash
 
-   .. group-tab:: Windows
+            .. include:: snippets/wsl-home-dir.rst
 
-      .. note::
 
-         Due to installation constraints, there is no way to use the native Nextstrain components on Windows directly. Follow steps for **WSL on Windows** if the native environment is desired, or use the **Docker**-based steps instead.
+         .. group-tab:: Ubuntu Linux
 
-      1. `Install Windows Subsystem for Linux (WSL) 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
+            .. note:: Steps for other Linux distributions (Debian, CentOS, RHEL, etc.) will be similar, though specific commands may vary slightly.
 
-         .. note:: You may have to restart your machine when configuring WSL.
+            1. Install Docker Engine using the standard Ubuntu package:
 
-      2. `Install Docker Desktop with WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
-      3. Install Miniconda:
+               .. code-block:: bash
 
-         a. Go to the `installation page <https://docs.conda.io/en/latest/miniconda.html>`_.
-         b. Scroll down to the **Latest Miniconda Installer Links** section and click the Windows platform link relevant to your machine.
-         c. Open the downloaded file and follow through installation prompts.
+                  sudo apt install docker.io
 
-      4. Open an Anaconda PowerShell Prompt, which can be found in the Start menu. Note that you should not use the *administrator* prompt.
-      5. Install Mamba on the ``base`` Conda environment:
+               .. note::
 
-         .. code-block:: powershell
+                  See `Docker's installation documentation <https://docs.docker.com/engine/install/ubuntu/>`__ for alternative installation methods.
 
-            conda install -n base -c conda-forge mamba --yes
-            conda activate base
+            2. Add your user to the `docker` group:
 
-      6. Create a Conda environment named ``nextstrain``:
+               .. code-block:: bash
 
-         .. include:: snippets/conda-create-powershell.rst
+                  sudo gpasswd --add $USER docker
 
-      7. Install the Nextstrain CLI:
+            3. Restart your machine.
+            4. Install the Nextstrain CLI:
 
-         .. code-block:: powershell
+               .. code-block:: bash
 
-            mamba install --yes nextstrain-cli
+                  curl -fsSL --proto '=https' https://nextstrain.org/install/cli/linux | bash
```

Comment:
This isn't the right URL, but the others are correct.

```suggestion
                  curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/linux | bash
```

Reply:
Ah, good catch! Fixed.